CSM resource evaluation filters (#20638)
DataDog/documentation: content/en/security/cloud_security_management/guide/resource_evaluation_filters.md

---
title: Use Filters to Exclude Resources from Evaluation
kind: guide
further_reading:
  - link: "/security/cloud_security_management/guide"
    tag: "Documentation"
    text: Cloud Security Management Guides
  - link: "/security/cloud_security_management/setup"
    tag: "Documentation"
    text: Setting Up Cloud Security Management
---

You can use resource tags to create filters that include or exclude resources from being evaluated by Cloud Security Management (CSM). The filters must be specified as a comma-separated list of `key:value` pairs.

**Notes**:

- Resource evaluation filters can only be used with hosts that are scanned by cloud integrations.

> - Tags must be applied directly to the resource. User tags added in Datadog are not taken into account by the filters. The only exception is for tags added on the integration tiles for AWS and Google Cloud Platform.

Reviewer: ```suggestion
- Tags must be applied directly to the resource. The filters do not take into account user tags added in Datadog. The only exception is for tags added on the integration tiles for AWS and Google Cloud Platform.
```
